pkg/cmd/roachtest/test_impl.go

```diff
@@ -297,9 +296,13 @@ func (t *testImpl) FailNow() {
 	panic(errTestFatal)
 }
 
+func (t *testImpl) Error(args ...interface{}) {
+	t.addFailure(argsToErr(1, args...))
+}
+
 // Errorf implements the TestingT interface.
 func (t *testImpl) Errorf(format string, args ...interface{}) {
-	t.addFailure(errors.NewWithDepthf(1, format, args...))
```

Comment:
Why remove the `WithDepthf`? The stack trace in the error (when formatted via `%+v`) should correspond to the caller of `Errorf`.